applications/GeoMechanicsApplication/tests/line_interface_elements/README.md

```diff
@@ -0,0 +1,21 @@
+# Line interface element tests with 3 + 3 nodes
+
+This folder contains integration tests for the line interface elements with Neumann and Dirichlet boundary conditions. 
+
+## Setup
+For these specific tests, two non-connected line interface elements with different orientations and 3 + 3 nodes are used, as seen in the schematic below. Note that the node pairs are actually at the same location, but have been separated for clarity. For more information on interface geometries, see the [geometry documentation](../../custom_geometries/README.md).
+
+![Schematic](Schematic.svg)
+
+The two tests (Dirichlet_single_stage and Neumann_single_stage) are performed in a single stage and a single time step. The Neumann test applies a normal + shear load to one side of the interfaces, while the Dirichlet test applies a fixed displacement. The other side of the interface is kept fixed.
```

Comment:
"tests" or "test cases"? Matter of preference, I prefer to mention them as test cases.

Reply:
I don't have a strong preference, so changed to test case(s)